manifests/params.pp

```diff
@@ -17,21 +17,28 @@
   $version = 'latest'
   $service = 'jenkins'
   $install_from_git = false
+  $install_from_pip = true
+  $install_from_pkg = false
   $git_revision     = 'master'
   $git_url          = 'https://git.openstack.org/openstack-infra/jenkins-job-builder'
 
   case $::osfamily {
     'RedHat', 'Amazon': {
-      $python_packages = [ 'python', 'python-devel', 'python-pip', 'python-argparse']
+      $python_packages = [ 'python', 'python-devel', 'python-pip', 'python-argparse', 'PyYAML']
```

Comment:
> We are installing them for all installation methods, so that should be covered.

Ah, sorry - I misread your changes, I thought you'd moved the `ensure_resources()` definition

> I've updated for argparse/el7 in #12

Awesome :)